data/hai/hai reveal 5 philosophers.txt

```diff
@@ -1011,84 +1011,127 @@ mission "Hai Reveal [B13] Freedom"
 				goto questions
 			label "cadmium"
+			action
+				set "Hai Reveal [B13] Freedom: label cadmium"
 			`	The Korath leader looks to the Hai, who responds, "Is that how you were led back here?"`
 			`	"Indirectly," you reply.`
-			`	The Hai leader continues, "Fascinating, we always wondered if it would pique someone's attention! Our Korath friends here brought a ship with some remarkable technology they call a "Systems Core". None of the surviving Korath who got here had the knowledge to replicate it exactly, but they were asked to try and many of them seem to have an uncanny knack for engineering. With their help, we managed to build some of our own in a fashion. Being not quite right, they work, but they fuse irrevocably with the ship, and some flaw in their function causes cadmium particulate, which is shed by the system maintenance processes, into the air where it is recirculated and ignored by the system intended to filter it out, since it's part of it."`
+			`	The Hai leader continues, "Fascinating, we always wondered if it would pique someone's attention! Our Korath friends here brought a ship with remarkable technology called a "Systems Core". None of the surviving Korath had the knowledge to replicate it, but they were asked to try and many of them have an uncanny knack for engineering. With their help, we built some of our own in a fashion. Being not quite right, they work, but they fuse irrevocably with the ship, and some flaw in their function causes cadmium particulate, which is shed by the system maintenance processes, into the air where it is recirculated and ignored by the system intended to filter it out, since it's part of it."`
 			`	The human leader adds, "We worked out how to reduce that effect to safe levels, but kept that information to ourselves in the hope that it would draw some attention. It tended to make the pirates sick more often than anyone else, which also seemed to mean that more of their victims escaped."`
 				goto questions
 			label home
+			action
+				set "Hai Reveal [B13] Freedom: label home"
 			`	The three leaders laugh. The Hai answers, "We are home. Most of us were born here, or have lived most of our adult lives here. This is a place of learning, where three races join as one to progress together. Even some of the more recent arrivals, several had few connections back in Hai space and have viewed this place as a grand new novelty. An adventure without compare. Some recent kidnapping victims wish to leave of course, and we will help you find them and send them home."`
 			`	The Korath speaks next, and the Hai translates. "Where my people came from, everyone was starving, barely subsisting. We had to raid for supplies, or slave away in bleak factory stations. This is a much better place for us."`
```

Comment:
```suggestion
				set "Hai Reveal [B13] Freedom: label home"
			`	The three leaders laugh. The Hai answers, "We are home. Most of us were born here, or have lived most of our adult lives here. This is a place of learning, where three races join as one to progress together. Even some of the more recent arrivals, several had few connections back in Hai space and have viewed this place as a grand new novelty. An adventure without compare. Some recent kidnapping victims wish to leave of course, and we will help you find them and send them home."`
			`	The Korath speaks next. "Where my people came from, everyone was starving, barely subsisting. We had to raid for supplies, or slave away in bleak factory stations. This is a much better place for us."`
```

Reply:
The text already explicitly states that some have learned the language. However, there is no indication yet that they are physically capable of making the required sounds. Showing Korath that can speak the language is a major story change.